docs/UserMenu.md

```diff
@@ -55,17 +55,19 @@ const avatar = <Avatar><SendIcon/></Avatar>;
-| ------------ | ------------------------------------------------ | ------------------ | -------- | ------- |
-| avatar       | MUI Avatar that displays                         | `Avatar`           | yes      |         |
-| classes      | Style overrides                                  | `UserMenuClasses`  | no       |         |
-| menu         | Custom MUI Menu displayed when Avatar is clicked | Material-UI `Menu` | no       |         |
-| menuGroups   | Groups of menu items that display                | `UserMenuGroups[]` | no       |         |
-| menuSubtitle | Subtitle shown when menu is open                 | `string`           | no       |         |
-| menuTitle    | Title shown when menu is open                    | `string`           | no       |         |
-| MenuProps    | Property overrides for the MUI Menu              | `MenuProps`        | no       |         |
-| onClose      | Function called when the menu is closed          | `Function`         | no       |         |
-| onOpen       | Function called when the menu is opened          | `Function`         | no       |         |
+| Prop Name          | Description                                                               | Type               | Required | Default |
+| ------------------ | ------------------------------------------------------------------------- | ------------------ | -------- | ------- |
+| avatar             | MUI Avatar that displays                                                  | `Avatar`           | yes      |         |
+| classes            | Style overrides                                                           | `UserMenuClasses`  | no       |         |
+| disableBottomSheet | Disable responsive bottom sheet menu                                      | `boolean`          | no       | false   |
+| menu               | Custom MUI Menu displayed when Avatar is clicked                          | Material-UI `Menu` | no       |         |
+| menuGroups         | Groups of menu items that display                                         | `UserMenuGroups[]` | no       |         |
+| menuSubtitle       | Subtitle shown when menu is open                                          | `string`           | no       |         |
+| menuTitle          | Title shown when menu is open                                             | `string`           | no       |         |
+| MenuProps          | Property overrides for the MUI Menu                                       | `MenuProps`        | no       |         |
+| onClose            | Function called when the menu is closed                                   | `Function`         | no       |         |
+| onOpen             | Function called when the menu is opened                                   | `Function`         | no       |         |
+| useBottomSheetAt   | Window pixel width at which the responsive bottom sheet menu is triggered | `number`           | no       | 600     |
```

Comment:
true story

Reply:
> Not a disagreement, just curious what you think

I thought the same thing as I was adding the disableBottomSheet prop but figured it's better to be explicit about disabling things. I could go either way as long as we document it appropriately.

Reply:
I would lean towards using the single prop...otherwise you've got two props that conflict with each other (i.e, if you didn't realize you had disabled on, you could be changing the usebotomsheetat to no avail).

You could cover this in the description, e.g.: '...menu is triggered (set to 0 to disable responsive behavior)'